web/src/components/navbar/NavSideBarUserSettings.tsx

```diff
@@ -28,38 +36,62 @@ export const NavSideBarUserSettings = (): ReactElement => {
           id="user-profile-header"
         >
           <div className="margin-y-2">
-            <h4 className="flex flex-align-center text-primary">
-              <Icon className="margin-right-1 usa-icon--size-3">account_circle</Icon>
-              <span>{userName}</span>
+            <h4
+              className={`flex flex-align-center text-${
+                state.isAuthenticated == "TRUE" ? "primary" : "base"
+              }`}
+            >
+              <Icon className="margin-right-1 usa-icon--size-3">
+                {state.isAuthenticated == "TRUE" ? "account_circle" : "help"}
+              </Icon>
+              <span>
+                {state.isAuthenticated == "TRUE" ? userName : Config.navigationDefaults.navBarGuestText}
+              </span>
```

Comment:
same comment as other bit

Reply:
maybe this should be a component, if it's shared between here and the other navbar component?